src/dapps/ens-manager-dapp/handlers/handlerPermanentName.js

```diff
@@ -179,7 +188,7 @@ export default class PermanentNameModule extends ENSManagerInterface {
         .call();
       return this.registrarControllerContract.methods
         .commit(commitment)
-        .send({ from: this.address });
+        .send({ from: this.address, gas: '500000' });
```

Comment:
@kvhnuke should i estimate gas here too ?

Reply:
sure, better to stay away from hardcoding gas unless it is absolutely necessary. for single txs you dont have to put gas since itll estimate it internally